qutebrowser/completion/models/base.py

```diff
@@ -44,6 +44,17 @@ class BaseCompletionModel(QStandardItemModel):
         COLUMN_WIDTHS: The width percentages of the columns used in the
                         completion view.
         DUMB_SORT: the dumb sorting used by the model
+
+    Attributes:
+        filtered_out_cache:
```

Comment:
Huh, that's kind of a bummer but I guess this is the best place for it right now. Just be aware that completion models are cached right now, so if you open, say, bookmark completion multiple times it is probably using the same instance every time (not sure if that affects anything here).